styles.css

```diff
@@ -154,6 +212,59 @@ input {
   font-weight: 300;
 }
 
+.timer-container,
+.completed-activity-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #46424D;
+  height: 80%;
+  border-radius: 4px;
+  box-shadow: 5px 10px 18px #26222D;
+  margin-bottom: 2%;
+  padding: 2%;
+  color: #FFF;
+  justify-content: space-around;
+  align-items: center;
+  overflow: hidden;
+}
+
+.start-timer.meditate {
+  border: 3px solid #C278FD;
+  background-color: #46424D;
+}
+
+.start-timer.exercise {
+  border: 3px solid #FD8078;
+  background-color: #46424D;
+}
+
+.start-timer.study {
+  border: 3px solid #B3FD78;
+  background-color: #46424D;
+}
+
+.timer-numbers {
```

Comment:
This looks much better than before. Alphabetizing our properties made a real difference in readability.